src/integrationTest/java/uk/gov/hmcts/reform/hmc/config/MessageProcessorIT.java

```diff
@@ -291,7 +294,8 @@ void shouldThrowMismatchOnRequestVersion() {
         assertEquals("Message of type HEARING_RESPONSE received", logsList.get(0).getMessage());
         assertEquals("Error processing message with Hearing id 2000000000 exception was "
                          + "Cannot find request version 10 for hearing 2000000000", logsList.get(1).getMessage());
-        assertEquals("Hearing id: 2000000000 updated to status Exception", logsList.get(2).getMessage());
+        assertEquals("Hearing id: 2000000000 Case ref: 9372710950276233 Service Code: TEST"
```

Comment:
Just a note to start using getFormattedMessage ...for when messages are updated to use variables for formatting the message